homeassistant/components/roborock/strings.json

```diff
@@ -267,5 +267,22 @@
         }
       }
     }
+  },
+  "exceptions": {
+    "command_failed": {
+      "message": "Error while calling {command} with {params}."
```

Comment:
This is user-facing messaging, do we need to send params?

Reply:
I guess not - typically this error is reached when the user uses the service Vacuum: Send Command, in which params will be right in front of them anyways.